tests/test_dataset.py

```diff
@@ -59,10 +61,10 @@ def testDirectories(self):
         """Verify that a Dataset reports the desired directory structure.
         """
         root = self._testbed.datasetRoot
-        self.assertEqual(self._testbed.rawLocation, os.path.join(root, 'raw'))
-        self.assertEqual(self._testbed.calibLocation, os.path.join(root, 'calib'))
-        self.assertEqual(self._testbed.templateLocation, os.path.join(root, 'templates'))
-        self.assertEqual(self._testbed.refcatsLocation, os.path.join(root, 'refcats'))
+        self.assertEqual(self._testbed.rawLocation, join(root, 'raw'))
+        self.assertEqual(self._testbed.calibLocation, join(root, 'calib'))
+        self.assertEqual(self._testbed.templateLocation, join(root, 'templates'))
+        self.assertEqual(self._testbed.refcatsLocation, join(root, 'refcats'))
```

Comment:
Would it be possible to revert the changes to this file? I don't see how they relate to the new metadata, and IMO `join` is much more confusing than `os.path.join` (at first glance it looks like a straight string join).

Reply:
More linting problems.  It was really hard to get these under the line limit.  I can't simply revert because travis fails as it was.

Reply:
That can't be right. I've been building that code with Travis for months, and `setup.cfg` uses the science pipelines limit (110 characters).

Reply:
Discussed this on Slack, but for the record: the only Travis errors I could find for this file were W504, which was fixed by #50. So it should be safe to drop this commit once that PR is in the history.

Reply:
OK, so travis has passed.